src/fortran/client.F90

```diff
@@ -135,74 +135,74 @@ module smartredis_client
 contains
 
 !> Initializes a new instance of a SmartRedis client
-subroutine initialize( this, cluster )
-  class(client_type) :: this
-  logical, optional :: cluster !< If true, client uses a database cluster (Default: .false.)
-
-  if (present(cluster)) this%cluster = cluster
-  this%client_ptr = c_constructor(this%cluster)
+subroutine initialize( client, cluster )
+  class(client_type), intent(inout) :: client    !< Receives the initialized client
+  logical, optional,  intent(in   ) :: cluster !< If true, client uses a database cluster (Default: .false.)
 
+  if (present(cluster)) client%cluster = cluster
+  client%client_ptr = c_constructor(client%cluster)
 end subroutine initialize
 
 !> A destructor for the SmartRedis client
-subroutine destructor( this )
-  class(client_type) :: this
+subroutine destructor( client )
```

Comment:
I thought ``client.F90`` changes were in a separate PR?  Did something get messed up on merge?

Reply:
I'm not sure how all the Fortran stuff got into this pull -- that's not even the final state. I'll back out all of these changes.